test/test_scatter_gather_ops.py

```diff
@@ -158,6 +179,46 @@ def test_scatter_add_mult_index_base(self, device, dtype):
         self.assertEqual(res0[0, :], m * torch.ones(n, device=device, dtype=dtype), atol=0, rtol=0)
         self.assertEqual(res1[:, 0], n * torch.ones(m, device=device, dtype=dtype), atol=0, rtol=0)
 
+    @onlyCPU
+    @toleranceOverride({torch.float16: tol(atol=1e-2, rtol=0),
```

Comment:
How did you determine these tolerances?

Reply:
For float16 this is copied from [the toleranceOverride decorator on test_scatter__reductions in this file](https://github.com/pytorch/pytorch/blob/22a3a17ea0d61660b53f6642fe22529a7e4e2649/test/test_scatter_gather_ops.py#L135). I copied this for bfloat16 but I'm not sure what tolerance should be used. How does one normally determine this

Reply:
There are default tolerances set, I'd first try running this without an override

Reply:
Removed these decorators, thanks!